.github/workflows/native-full_marian-ubuntu.yml

```diff
@@ -2,9 +2,9 @@ name: Native (Full Marian) Ubuntu
 
 on:
   push:
-    branches: [ main, ci-sandbox ]
+    branches: [ main, ci-test ]
```

Comment:
My intentions of `ci-sandbox` is to be force-pushed for this sort of situations, when the only way is to debug is push to github runner without creating a mess of commit history and notification spam. I will make the update eventually in a separate PR and update somewhere.